docs/en/contributor/Building-Alluxio-From-Source.md

```diff
@@ -25,14 +27,22 @@ $ git clone git://github.com/alluxio/alluxio.git
 $ cd alluxio
 ```
 
-Optionally, you can build a particular version of Alluxio, for example {{site.ALLUXIO_RELEASED_VERSION}}.
-Otherwise, this will build the master branch of the source code.
+Optionally, you may build a particular version of Alluxio.
```

Comment:
Could make this part more direct saying 

"This will check out the master branch by default. To build a particular version of Alluxio, check out the tag for that version. For example, to check out version {{site.ALLUXIO_RELEASED_VERSION}}, run"